src/components/MomentsSection.jsx

```diff
@@ -0,0 +1,107 @@
+import PropTypes from "prop-types";
+import { useState } from "react";
+import { IoIosArrowDropdown, IoIosArrowDropup } from "react-icons/io";
+
+const MomentsSection = ({
+  className,
+  bgColor,
+  hoverBgColor,
+  textColor,
```

Comment:
can we also remove this or make this default `text-white` like in the summary section, this color doesn't change between conferences and I don't see it being anything other than white in any variation